decred/decred/crypto/secp256k1/curve.py

```diff
@@ -166,7 +166,7 @@ def __init__(
         self.H = H
         self.q = q
         self.byteSize = byteSize
-        self.lamda = lamda
+        self.lambda_ = lambda_
```

Comment:
In other cases I've added some qualifying prefix to the reserved built-in, like koblitzLambda. I don't know if one way or the other is better, I do think that just misspelling the word is confusing, but whatever we do, I hope to use the same pattern throughout.

Reply:
If a prefix is needed just to avoid a conflict and repeats info found elsewhere, I find it a bit redundant. The underscore suffix convention works and is more readable. (Not an underscore *prefix* though, because that has a different meaning.)